modules/storage/storage-account/main.bicep

```diff
@@ -214,11 +96,108 @@ module storageRbac 'modules/rbac.bicep' = [for (roleAssignment, index) in roleAs
     blobName: blobName
     containerName: blobContainerName
   }
-}]    
+}]
 
 
 @description('The name of the Storage Account resource')
 output name string = name
 
 @description('The ID of the Storage Account. Use this ID to reference the Storage Account in other Azure resource deployments.')
 output id string = storageAccount.id
+
+
+@description('The properties of a storage account’s Blob service.')
+type blobServiceProperties = {
+  changeFeed: changeFeed?
+  containerDeleteRetentionPolicy: containerDeleteRetentionPolicy?
+  cors: cors?
+  deleteRetentionPolicy: deleteRetentionPolicy?
+  isVersioningEnabled: isVersioningEnabled?
+  lastAccessTimeTrackingPolicy: lastAccessTimeTrackingPolicy?
+  restorePolicy: restorePolicy?
+}
+
+@description('The blob service properties for change feed events.')
+type changeFeed = {
+  enabled: bool
+  @minValue(1)
+  @maxValue(146000)
+  retentionInDays: int?
+}
+
+@description('The blob service properties for container soft delete.')
+type containerDeleteRetentionPolicy = {
+  allowPermanentDelete: bool
+  @minValue(1)
+  @maxValue(365)
+  days: int?
+  enabled: bool
+}
+
+@description('Specifies CORS rules for the Blob service. You can include up to five CorsRule elements in the request. If no CorsRule elements are included in the request body, all CORS rules will be deleted, and CORS will be disabled for the Blob service.')
+type cors = {
+  corsRules: [
+    {
+      allowedHeaders: [
+        'string'
+      ]
```

Comment:
Should these be `string[]` instead? The current type declaration will only permit the value `['string']`.